cmd/vic-machine/create/create.go

```diff
@@ -1292,61 +1312,80 @@ func logArguments(cliContext *cli.Context) {
 				continue
 			}
 			url.User = nil
-			log.Debugf("--target=%s", url.String())
+			flag := fmt.Sprintf("--target=%s", url.String())
+			log.Debug(flag)
+			args = append(args, flag)
```

Comment:
could we compare the default value here before actually log out parameter and append command?
I'm asking this is because with our command definition, cli will fit some default value into the input. If we could filter out those, this solution would be perfect.